packages/jest-jasmine2/src/each.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
This should have the full license header, methinks.

```
/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */
```

Reply:
not sure about that one, though...

Reply:
Yeah I wasn't sure either - I can add it to be on the safe side

Reply:
Done in 30ebfb5